### ISSUE TITLE
 Add guide for installing Apache APISIX on KubeSphere

### ISSUE THREAD

**Issue by moonming** [kubesphere/website#1004]
@lilien1010 shared a related share on Apache APISIX DevCon, can it be synchronized to the document?
thx

**Comment by lilien1010** [comment]
more like a case study for users
`https://kubesphere.com.cn/case/`

**Comment by FeynmanZhou** [comment]
> more like a case study for users
> `https://kubesphere.com.cn/case/`
> But already I post it in KubeSphere community,

I assume there should a deployment guide of installing Apache APISIX on KubeSphere. See Install [Install Gitlab on KubeSphere](https://kubesphere.io/docs/application-store/external-apps/gitlab-app/) as an example.

**Comment by lilien1010** [comment]
Can we name APISIX as `Built-in Applications` now?
which I think we can post in this directory `/docs/application-store/built-in-apps/apisix-app/`.
sounds good to you ? @FeynmanZhou

**Comment by FeynmanZhou** [comment]
> Can we name APISIX as `Built-in Applications` now?
> which I think we can post in this directory `/docs/application-store/built-in-apps/apisix-app/`.
> sounds good to you ? @FeynmanZhou

Yes, but I think it can be released as a built-in app with v3.1.0 out. For v3.0.0, users can install APISIX on KubeSphere by importing a Helm repo.
